pom.xml

```diff
@@ -197,6 +197,11 @@
         <artifactId>jenkins-test-harness-tools</artifactId>
         <version>2.2</version>
       </dependency>
+      <dependency>
+        <groupId>org.jenkins-ci.main</groupId>
+        <artifactId>jenkins-test-harness</artifactId>
+        <version>2.39</version>
```

Comment:
Rather override `jenkins-test-harness.version`.

Reply:
I forgot that was a thing.